#### Урок 4. Непрерывные случайные величины. Функция распределения и плотность распределения вероятностей. Равномерное и нормальное распределение. Центральная предельная теорема

In [49]:
from math import sqrt, pi, exp

### 1. Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

$$M(X) = \frac{a + b}{2}$$

In [50]:
m = (200 + 800) / 2
print(f'Среднее значение = {m}')

Среднее значение = 500.0



$$D(X) = \frac{(b - a)^2}{12}$$

In [51]:
d = (800 - 200)**2 / 12
print(f'Дисперсия = {d}')

Дисперсия = 30000.0


**Ответ:** 

Среднее значение = 500.0

Дисперсия = 30000.0

### 2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.

0.2 = (b - 0.5)**2 / 12

Раскрываем скобки, приводим в стандартный вид:

$$(b^2 - b + 0.25) / 12 = 0.2$$

$$b^2 - b - 2.15 = 0$$

Решаем через дискриминант $$(D = b^2 - 4ac = 1 + 4*2.15 = 9.6)$$

$$b1 = (-b + \sqrt d )/ 2a = (1 + \sqrt 9.6) / 2$$

$$b2 = (-b - \sqrt d) / 2a = (1 - \sqrt 9.6) / 2$$

In [52]:
b1 = (1 - (9.6)**0.5) / 2
b1

-1.0491933384829668

In [53]:
b2 = (1 + (9.6)**0.5) / 2 
b2

2.049193338482967

По условию, 0,5 - лева граница, значит b > 0.5, значит из двух корней верный = 2.05

$$M(X) = \frac{a + b}{2}$$

In [54]:
m = (0.5 + 2.05) / 2
print(f'Среднее значение = {m}')

Среднее значение = 1.275


**Ответ**

b = 2.05

M(X) = 1.275

### 3. Непрерывная случайная величина X распределена нормально и задана плотностью распределения


$$f(x) = \frac{1}{4 \sqrt{2 \pi}} e ^ {- \frac{(x + 2)^2}{32}},$$

Найдите:

а). M(X)

б). D(X)

в). std(X) (среднее квадратичное отклонение)

$$f(x) = \frac{1}{\sigma \sqrt{2 \pi}} e ^ {- \frac{(x - a)^2}{2 \sigma^2}},$$

Исходя из формулы: сигма = 4 (значит дисперсия  = 16), среднее = -2.

**Ответ:**

M(X) = -2

D(X) = 16

std(X) = 4

### 4. Рост взрослого населения города X имеет нормальное распределение.

### Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.

### Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:

а). больше 182 см

б). больше 190 см

в). от 166 см до 190 см

г). от 166 см до 182 см

д). от 158 см до 190 см

е). не выше 150 см или не ниже 190 см

ё). не выше 150 см или не ниже 198 см

ж). ниже 166 см.

$$ Z = \frac{(x - m(x))}{sigma}  $$

In [55]:
z1 = (182 - 174) / 8
print(f'Z-значение = {z1}')

Z-значение = 1.0


По таблице, рост выше 182 см это больше, чем у 84,13% всех наблюдений. Т.о. вероятность такого роста 15,87% (100 - 84,13)

In [56]:
z2 = (190 - 174) / 8
z2

2.0

По таблице, это больше, чем у 97,725% наблюдений. 

In [57]:
100 - 97.725

2.2750000000000057

2.275% вероятности.

In [58]:
z3 = (166 - 174) / 8
z3

-1.0

Это больше, чем у 15,87%.


In [59]:
100 - (2.2175 + 15.87)

81.9125

Т.о. рост между 166 и 190 см это 81.9125% всех наблюдений.


Рост между 166 и 182 см:

In [60]:
100 - (15.87*2)

68.26

In [61]:
z4 = (158 - 174) / 8
z4

-2.0

Рост между 158 и 190 см:

In [62]:
100 - (2.275*2)

95.45

Это 95,45% всех наблюдений

In [64]:
z5 = (150 - 174) / 8
z5

-3.0

Z(-3) = 0.00135 = 0.135%

In [65]:
15.87 + 0.135

16.005

Вероятность быть не выше 150 и не ниже 190 = 16.005

In [66]:
z6 = (198 - 174) / 8
z6

3.0

In [67]:
0.135 + 0.135

0.27

Вероятность быть не выше 150 и не ниже 198 = 0,27

In [68]:
z7 = (166 - 174) / 8
z7

-1.0

Вероятность быть ниже 166 = 15.87%

**Ответ**:

а). больше 182 см = 15,87%

б). больше 190 см = 2,275%

в). от 166 см до 190 см = 81,9125%

г). от 166 см до 182 см = 68,26%

д). от 158 см до 190 см = 95,45%

е). не выше 150 см или не ниже 190 см = 16,005%

ё). не выше 150 см или не ниже 198 см = 0,27%

ж). ниже 166 см. = 15,87%

### 5. На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [69]:
sigma = sqrt(25)
sigma

5.0

In [70]:
Z = (190 - 178) / 5
Z

2.4

**Ответ:**

На 2,4 сигмы.